### DISCUSSION TITLE
Make available outside of Vite

### DISCUSSION THREAD

**Discussion by danimalweb** [cssninjaStudio/unplugin-fonts#12] [Ideas]
I love what you've done here. We use Vite in a non-standard way (no entry point to inject the tags).

Been looking for a solid replacement for the WebFont loader for a number of years and we currently run a forked version to take advantage of Google Fonts API v2.

This would be awesome as a standalone import using regular JS (or TS) to inject the required tags and also the generation of the custom CSS (maybe via an inject of a style block in the head).

I didn't see a License associated with your repo. I'm considering forking it and seeing if I can unwrap the need for it to be dependent on a Vite-only setup.

Interested in hearing your thoughts on this. Great work!

**Comment by danimalweb** [comment]
@stafyniaksacha Any thoughts on the above?

**Reply by stafyniaksacha** [reply]
Hey! Sorry for the delay to answer!

I got the same issue while using Nuxt3 (related to: https://github.com/stafyniaksacha/vite-plugin-fonts/issues/9) that seem to be related to having an entry point.

Can you explain how you want to use it?

Note that I've set the license to MIT, I forget to set it, feel free to fork (and submit a PR 😄)

**Reply by danimalweb** [reply]
@stafyniaksacha I love the way you are loading these fonts using best practices, but not the dependency on having to have an entry point for Vite (as we don't have an entry point to inject into).

It would be ideal (imo) if the dependency of Vite could be removed and this is used as a separate npm module with the config in a standard JS object and the script tags injected into the head using pure JS and not the Vite function. Once I've put a couple of in-progress projects to bed I might take a bash at this now I know it's an MIT license. I'll post back here if/when I make some progress.
